app/views/collection_drafts/forms/fields/_instrument_child_fields.html.erb

```diff
@@ -2,15 +2,17 @@
   <div class="row">
     <div class="col-6">
       <!-- Short Name [r] -->
-      <%= mmt_text_field(
+      <%= mmt_select(
         name: 'short_name',
-        title: 'ShortName',
+        title: 'Short Name',
         prefix: name_prefix,
         value: instrument_child['ShortName'],
+        options: @instruments.map { |instrument| [instrument[:short_name], instrument[:short_name], { 'data-long-name' => instrument[:long_name] }] },
         help: 'definitions/PlatformShortNameType',
+        classes: 'select2-select instrument-short-name-select',
         validate: true,
-        required: true
-       ) %>
+        always_required: true
```

Comment:
We just tested with this field reverted to `required: true`, and in the instrument fields as well. That seemed closer to the expected behavior, but the required icon for the instrument and instrument child short name does not always go away when expected, sometimes it needed an additional blur off of the long name as well.